client/src/components/Weather/index.js

```diff
@@ -53,61 +52,59 @@ export default function getWeather() {
 
 
     return (
-        <>           
-         <button
-         id="emojid"
-            style={{
-                width: "100px",
-                borderRadius: "3px",
-                position:  "absolute",
-                marginTop: "145%",
-                marginLeft: "3%"
-
-            }}
-            onClick={e => {
-                e.preventDefault();
-                getWeather();
-                document.getElementById("weather-container").style.display = "block";
-            }}
-
-            className="emoji btn btn-large waves-effect waves-light hoverable blue accent-3">
-            ⛈️
+        <>
+            <button
+                id="emojid"
+                style={{
+                    width: "100px",
+                    borderRadius: "3px",
+                    position: "absolute",
+                    marginTop: "145%",
+                    marginLeft: "3%"
+
+                }}
+                onClick={e => {
+                    e.preventDefault();
+                    getWeather();
+                    document.getElementById("weather-container").style.display = "block";
+                }}
+                className="emoji btn btn-large waves-effect waves-light hoverable blue accent-3">
+                {/* ⛈️ */}
```

Comment:
This change might be an issue

it was commented out to make a warning go away. original should honestly be fine